autoware.repos

```diff
@@ -0,0 +1,53 @@
+repositories:
+  autoware/autoware:
+    type: git
+    url: git@github.com:autowarefoundation/autoware.git
+    version: master
+  autoware/core_control:
+    type: git
+    url: git@github.com:autowarefoundation/core_control.git
+    version: master
+  autoware/core_perception:
+    type: git
+    url: git@github.com:autowarefoundation/core_perception.git
+    version: master
+  autoware/core_planning:
+    type: git
+    url: git@github.com:autowarefoundation/core_planning.git
+    version: master
+  autoware/documentation:
+    type: git
+    url: git@github.com:autowarefoundation/documentation.git
+    version: master
+  autoware/messages:
+    type: git
+    url: git@github.com:autowarefoundation/messages.git
+    version: master
+  autoware/simulation:
+    type: git
+    url: git@github.com:autowarefoundation/simulation.git
+    version: master
+  autoware/utilities:
+    type: git
+    url: git@github.com:autowarefoundation/utilities.git
+    version: master
+  autoware/visualisation:
+    type: git
+    url: git@github.com:autowarefoundation/visualisation.git
+    version: master
+  drivers:
+    type: git
+    url: git@github.com:autowarefoundation/drivers.git
+    version: master
+  citysim:
+    type: git
+    url: git@github.com:CPFL/osrf_citysim.git
+    version: 53b8483d098999298498854f3795a9f01c5e8828
+  car_demo:
+    type: git
+    url: git@github.com:CPFL/car_demo.git
+    version: e364448fad421cb6244c9f828f978d8d877dcbf9
+  lgsvl/lgsvl_msgs:
```

Comment:
The lgvsl_msgs package was released into the ROS buildfarm (https://index.ros.org/r/lgsvl_msgs/github-lgsvl-lgsvl_msgs/), so we no longer need to add it here.